### PR TITLE
Feature/forms

### PR DESCRIPTION
**Pull request by Gracetemitope**
This is a basic project that consists of building a simple form to obtain a username, email, and password, and store that data with Active Record.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to @user
```
Comment (GzimAsani) [inline app/controllers/users_controller.rb]: Instead of redirecting to @user please redirect it to edit_user_path

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+This is a basic project that consists of building a simple form to obtain a username, email, and password, and store that data with Active Record.
+
+# Built With
+- Ruby on Rails
+- SQLite
+- Rubocop
+
+To Run: 
+- Clone this repository.
```
Comment (GzimAsani) [inline README.md]: Specify how to clone the repo

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

### Hello **TEAM**, great job so far :wave:, you have met most of the project requirements, however there are few changes that have to be made, please check them below:
- [x] We recommend the form uses form_for rather than form_with
- [x] Also, add and comment out forms that use the normal form and form_tag
- [x] Fix the linting issues with the rubocop when I run the command rubocop . [Follow this guide for help](https://github.com/microverseinc/linters-config/tree/master/ror)

### Please implement these changes and ask for another code review whenever you are ready

_**If you have any question related to the required changes feel free to text me on chat @GzimAsani**_

HAPPY CODING :computer: :desktop_computer:

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑🛑**
Hello team,
Great implementation of the required changes.
I like the pace you are working on👍👍

Kindly fix one final change for me

[REQUIRED]: In the form action, instead of /users, use users_path.
[REQUIRED]: Also be more descriptive in the readme file on how to install the project. Like, include the link of your app to clone. This is because everyone is not used to cloning apps.

Kindly make the requested changes and ask for review👍
Happy coding💻💻💻

**Review by MahmoudBakr23 — APPROVED** [review]
# **Approved ✔✔**

<hr>

Hello there, great work you've done so far on this project and the requested changes earlier. Congratulations! 🎉✨

You may now merge to the development branch and submit. 👌
Good luck with the next, keep up the great work, and happy coding. 👨‍💻💻

**Pull request merged**